ref_ts/views.py

```diff
@@ -209,11 +209,12 @@ def create_ref_time_series(request, *args, **kwargs):
         context = {'resource_creation_error': "Error: failed to create resource." }
         return render_to_response('pages/create-ref-time-series.html', context, context_instance=RequestContext(request))
 
-@api_view(['GET'])
+
 def download_refts_resource_files(request, shortkey, *args, **kwargs):
```

Comment:
This download is always for downloading a bag right? In that case may be better to rename the function name something like ```download_refts_resource_bag```